lessons/pydata/pandas_groupby/index.ipynb

```diff
@@ -0,0 +1,3297 @@
+{
```

Comment:
<p>Jo, tady to začíná být divoké :-D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>